src/components/HeroText/HeroText.module.css

```diff
@@ -1,172 +1,162 @@
 .position {
-  margin-top: 100px;
-  margin-left: 400px;
+    margin-top: 100px;
+    margin-left: 400px;
 }
 
 .hero {
-  font-size: calc(1em + 2vw);
-  font-family: 'Josefin Sans', sans-serif;
-  /* max-width: 50vw; */
+    font-size: calc(1em + 2vw);
+    font-family: 'Josefin Sans', sans-serif;
+    /* max-width: 50vw; */
 }
 
 .bodyText {
-  /* max-width: 35rem; */
-  position: relative;
-  line-height:1.8;
-  font-size: calc(0.8em + 0.3vw);
-  font-family: 'Poppins';
-  max-width: 40vw;
+    /* max-width: 35rem; */
+    position: relative;
+    line-height: 1.8;
+    font-size: calc(0.8em + 0.3vw);
+    font-family: 'Poppins';
+    max-width: 40vw;
 }
 
 .f {
-  position: relative;
-  width: calc(18px + 1vw);
+    position: relative;
+    width: calc(18px + 1vw);
 }
 
-.left{
-  position: relative;
-  left: 70px;
-  top: 40px;
-  width: calc(18px + 1vw);
-  transform: scaleY(-1) rotate(315deg);
+.left {
+    position: relative;
+    left: 70px;
+    top: 40px;
+    width: calc(18px + 1vw);
+    transform: scaleY(-1) rotate(315deg);
 }
 
-.right{
-  position: relative;
-  /* left: calc(2em + 10vw); */
-  top: calc(14em + 0.1vw);
-  width: calc(18px + 1vw);
-  transform: rotate(45deg);
+.right {
+    position: relative;
+    /* left: calc(2em + 10vw); */
+    top: calc(14em + 0.1vw);
+    width: calc(18px + 1vw);
+    transform: rotate(45deg);
```

Comment:
files that are not part of "burger-menu" should be committed in a separate Pull request imo